# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

std_msgs/Header header
int32 waypoint
duration waypoint_time
duration expected_waypoint_time
time waypoint_start_time
time last_tf_update
geometry_msgs/Pose2D goal_pose
geometry_msgs/Pose2D robot_pose
geometry_msgs/Twist command

